src/components/network-status/styles.css

```diff
@@ -0,0 +1,18 @@
+.network-status-message {
+  align-items: center;
+  bottom: 0;
+  height: 50px;
+  justify-content: center;
+  pointer-events: none;
+  position: absolute;
+  width: 100vw;
+  z-index: 9999;
+}
+
+.network-status-message.offline {
+  background-color: rgba(255, 0, 0, 0.65);
+}
+
+.network-status-message.online {
+  background-color: rgba(0, 255, 0, 0.55);
+}
```

Comment:
Awesome work on the hook! Thank you 🙏

One more thing I noticed is the use of custom css. To be consistent with the rest of the project, let's use Bootstrap classes and be free from this css file? For example, we want the status message stick to the bottom, which would be accomplished by adding "fixed-bottom" to the div element's className.

Reply:
@kumikokashii Ok, I'll use Bootstrap